package.json

```diff
@@ -11,12 +11,17 @@
   },
   "dependencies": {
     "@astrojs/check": "^0.8.3",
```

Comment:
What's the point of using a static site generator like astro, only to then pull in react as a client-side dependency?